lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -89,155 +134,122 @@ class StorageEnclosureConfig(Command):
         }
     }
 
+    def __init__(self):
+        super().__init__()
+        if enc_file_path.exists():
+            if Path(enc_file_path).stat().st_size != 0:
+                with open(enc_file_path, "r") as file:
+                    enclosure_id = file.read().replace('\n', '')
```

Comment:
Something seems to have been missed here.

Reply:
Here I'm just checking if /etc/enclosure_id file is present and it is not empty and then storing the file content(i.e value) in the enclosure_id variable.